src/components/JobConfiguration.vue

```diff
@@ -94,7 +94,7 @@
       <ion-icon slot="start" :icon="timeOutline" />
       {{ $t("History") }}
     </ion-item>
-    <ion-item :disabled="!hasPermission(Actions.APP_JOB_UPDATE)" @click="runNow(currentJob)" button>
+    <ion-item :disabled="!hasPermission(Actions.APP_JOB_UPDATE)" @click="openJobCustomParameterModal(true)" button>
```

Comment:
Handle case of opening parameter model on all the instances from where we can initiate a runNow call.